src/com/rcv/ResultsWriter.java

```diff
@@ -479,6 +476,20 @@ private int addHeaderRows(
     DateFormat dateFormat = new SimpleDateFormat("M/d/yyyy");
     // string for formatted date
     String dateString = dateFormat.format(new Date());
+    // string indicating single- or multi-winner
+    String electionType = config.numberOfWinners() > 1 ? "multi-winner" : "single-winner";
+    // number reflecting threshold percentage to win
+    BigDecimal thresholdPercentage = config.divide(
+        new BigDecimal(100),
+        new BigDecimal(config.numberOfWinners() + 1)
+    );
+    // string for threshold percentage
+    String thresholdString = thresholdPercentage.toString() + "%";
+    // formula string
+    String formula = config.numberOfWinners() > 1 ?
```

Comment:
Maybe this should instead check electionType, rather than duplicate the same logic here?

Reply:
I think checking electionType would be a bit more fragile, because if I changed that string, I'd have to change it here also.